README.md

```diff
@@ -10,3 +10,12 @@ This repository contains
      * configuration file parsing
      * argument parsing
      * RPCs
+  3. The following CLI tools for debugging:
+     * lib/channel_helper.exe -- a channel passing helper CLI
+     * [A] lib_test/test.exe -- a test suite built from tests in lib_test
```

Comment:
Don't put the `test.exe`

Reply:
You're right, it's not a CLI tool. Should I remove `example.exe` too?